db/seeds/development.rb

```diff
@@ -40,3 +39,13 @@
                access_token_audience: 'va.gov',
                logout_redirect_uri: 'http://localhost:3001',
                refresh_token_duration: SignIn::Constants::RefreshToken::VALIDITY_LENGTH_SHORT_MINUTES)
+
+# Create Config for example external client
+vatemplate = SignIn::ClientConfig.find_or_initialize_by(client_id: 'vatemplate')
```

Comment:
Do you mind throwing in another client config for `API` authentication? I made the sample app workable with both `api` and `cookie`. So could we name this `cookietemplate` or something?